stdlib/sql/to_privates_test.go

```diff
@@ -129,8 +132,8 @@ func TestSqliteTranslation(t *testing.T) {
 			t.Log(cmp.Diff(nil, err))
 			t.Fail()
 		}
-		if !cmp.Equal(columnLabel+" "+dbTypeString, v) {
-			t.Log(cmp.Diff(columnLabel+" "+dbTypeString, v))
+		if !cmp.Equal(quote(columnLabel)+" "+dbTypeString, v) {
+			t.Log(cmp.Diff(quote(columnLabel)+" "+dbTypeString, v))
 			t.Fail()
```

Comment:
I prefer a more concise style that I've seen in our Go repositories:
```Go
if diff := cmp.Diff(quote(columnLabel)+" "+dbTypeString, v); diff != "" {
  t.Errorf("quoted ID does not match: (-want/+got):\n%s", diff)
}
```
But this is a matter of personal style.